src/components/skipNavLink.js

```diff
@@ -0,0 +1,42 @@
+import { div, h } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+export const SkipNavLink = Utils.forwardRefWithName('SkipNavLink', (props, ref) => {
+  return h(Link, {
+    as: 'a',
+    href: '#',
+    className: 'reveal-on-focus',
+    style: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      background: 'white',
+      color: colors.accent(),
+      border: '1px solid black',
+      borderColor: colors.accent(),
+      padding: '1em',
+      boxShadow: 'rgba(0, 0, 0, 0.5) 0px 0px 4px 0px',
+      zIndex: 9998
```

Comment:
We trust your judgment that this is the best way to do this, but would you also please add a line comment?

Reply:
Sure, make sense. I mostly just need to ensure that the button and outline both appear on top of anything else defined after them in the DOM